test/performance/odp_bench_buffer.c

```diff
@@ -870,6 +915,13 @@ int main(int argc, char *argv[])
 
 	ret = gbl_args->suite.retval;
 
+	if (common_options.is_export) {
```

Comment:
Probably it doesn't make much sense to try to export results if the test returned failure (`ret != 0`). Same comment for all applications.